bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/AudioFormat.java

```diff
@@ -173,6 +178,39 @@ public AudioFormat(@Nullable String container, @Nullable String codec, @Nullable
         this.bitDepth = bitDepth;
         this.bitRate = bitRate;
         this.frequency = frequency;
+        this.channels = 1;
```

Comment:
When defining a new constructor you should reuse it here by calling `super(...)` and passing all given parameters to it.

Reply:
you mean this(...) and not super(...) ?
If so, then ok, done.

Reply:
Oh man, sometimes I can be really, really stupid ! Thanks, done.